### DISCUSSION TITLE
How to read the data from internal RV sensor ?

### DISCUSSION THREAD

**Discussion by netbuyer** [Seagate/openSeaChest#86] [Q&A]
Hello I have a Seagate enterprise HDD, per my understands it has RV sensor. can I read the data from RV sensor by using OpenSeaChest_Log ? thanks.

**Comment by vonericsen** [comment]
Hi @netbuyer,

The RV sensor is used by the drive firmware and readings are not available through SMART or other logs.

**Reply by netbuyer** [reply]
Hi @vonericsen , we found a item : "RV Absolute Mean" in the log when we test OpenSeaChest on a Seagate enterprise HDD, can I understand what it mean is ?

**Reply by vonericsen** [reply]
I forgot that this was part of the FARM output. There is both RV abs mean and Max RV abs mean as part of FARM. These are readings from the RV sensor on the drive, but it is a product unique reported value that must be converted by Seagate internally to understand. Seagate does not share this conversion.
